src/main/resources/twilio.yaml

```diff
@@ -1,3 +1,4 @@
-sid: 'nope'
-token: 'slope'
-shortCode: ""
\ No newline at end of file
+sid: "nope"
+token: "slope"
+shortCode: ""
+callbackBaseUrl: "https://api.openlattice.com"
```

Comment:
should this be localhost? b/c we use local file for local testing?

Reply:
It can't be localhost since twilio will use this on their end to send messages over the network, so I usually use ngrok to test locally. I can just leave it blank though to avoid confusion

Reply:
Haaaaa, duuuhh 🤦 ty